src/crimson/common/errorator.h

```diff
@@ -859,6 +860,44 @@ struct errorator {
     return make_ready_future<>();
   }
 
+  template <typename Iterator, typename Func>
+  static inline errorator<AllowedErrors...>::future<>
+  parallel_for_each(Iterator begin, Iterator end, Func&& func) noexcept {
```

Comment:
could you extract this change into its own PR, and add a tiny test in `src/test/crimson/test_errorator.cc` ? i think probably @athanatos is also adding helpers like this to errorator and/or interruptible future.

Reply:
This seems orthogonal to what I'm doing.

Reply:
> could you extract this change into its own PR, and add a tiny test in `src/test/crimson/test_errorator.cc` ? i think probably @athanatos is also adding helpers like this to errorator and/or interruptible future.

please take a look at [#41848](https://github.com/ceph/ceph/pull/41848)